tardis/model/tests/test_config_model.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import pandas as pd
+import tardis
+import os
+from tardis.io.config_reader import Configuration
+from tardis.model import Radial1DModel
+import pytest
+
+DATA_PATH = os.path.join(tardis.__path__[0],'model','tests','data')
+
+@pytest.fixture(scope='module')
+def config_hard_coded():
```

Comment:
More descriptive function name and include docstring describing its purpose for these tests.